netci.groovy

```diff
@@ -470,6 +414,66 @@ branchList.each { branchName ->
     }
 }
 
+// adds steps to a job to download coreclr artifacts and corefx artifacts and invoke run-test.sh.
```

Comment:
We should have such a run somewhere, with a badge that people pay attention to.  I don't think it matters whether it's in the coreclr repo or in the corefx repo, but we do need such runs.  We already know of at least one bug that went unnoticed for a while but would have been caught immediately had we had this.